libs/ledger-live-common/src/families/ethereum/synchronisation.ts

```diff
@@ -586,12 +586,22 @@ const fetchAllTransactions = async (api: API, address, blockHeight) => {
   let getTransactionsResult: Tx[];
   let txs: Tx[] = [];
   let maxIteration = 20; // safe limit
+  let lastCallTxs: Tx[] | undefined;
```

Comment:
Just an optim proposal: maybe you don't need to store the full txs array but just the hash of the last one?

Reply:
Indeed !

Reply:
Done :)